src/olympia/reviewers/templatetags/code_manager_tags.py

```diff
@@ -0,0 +1,19 @@
+from django import template
```

Comment:
I don't think we need the `_tags` suffix on the filename, the file lives inside `templatetags/` already.